tests/operators/test_local.py

```diff
@@ -1457,6 +1457,7 @@ def test_async_execution_without_start_task(mock_read_sql, mock_bq_execute, monk
     mock_bq_execute.assert_called_once()
 
 
+@pytest.mark.integration
```

Comment:
do we know what part of this test needs an integration? i am hoping if we could mock something more that maybe missing to be mocked and keep this as a unit test? Maybe we could add a TODO for this to recheck later.

Reply:
This will need a little bit more work. This is the new ticket: #1608